web/cobrands/bexley/base.scss

```diff
@@ -105,3 +105,21 @@ ol.big-numbers > li:before {
 #postcodeForm .form-hint {
 //    color: $white;
 }
+
+.dropzone {
```

Comment:
Rather than having a large number of council specific overrides like this, which leads to larger CSS as the defaults are also included, it would be better to create any new $dropzone SASS variables for any cobrand that needs them, which would then make all these overrides unnecessary. For example, you could have a `$dropzone-border-colour` variable, defaulting to `$primary`, and any cobrand that needs something different can set `$dropzone-border-colour` in its colours.scss and that's all they have to do, no new actual CSS.

Relatedly, $dropzone-link-colour is now only used to set the dropzone-background-colour--awakened which doesn't feel ideal, could it be used as the button link or background colour, or dropped entirely.

Reply:
Looking at what's used in the overrides, it would look like almost all could be covered by having additional variables `$dropzone-border-colour`, `$dropzone-button-text` (or repurpose link-colour), `$dropzone-button-background` and `$dropzone-button-border`, I think?